app/service/cloud_watch_service.rb

```diff
@@ -0,0 +1,37 @@
+class CloudWatchService
+  def self.week_submissions(form_id:)
+    region = "eu-west-2"
+    a_week = 604_800
+    cloudwatch_client = Aws::CloudWatch::Client.new(region:)
+
+    response = cloudwatch_client.get_metric_statistics({
+      metric_name: "submitted",
+      namespace: metric_namespace.to_s,
+      dimensions: [
+        {
+          name: "form_id",
+          value: form_id.to_s,
+        },
+      ],
+      start_time: one_week_ago,
+      end_time: start_of_today,
+      period: a_week,
+      statistics: %w[Sum],
+      unit: "Count",
+    })
+
+    response.datapoints[0]&.sum.to_i || 0
+  end
+
+  def self.metric_namespace
+    "forms/#{Settings.forms_env}".downcase
+  end
+
+  def self.one_week_ago
+    Time.zone.now.midnight - 7.days
```

Comment:
I'm not sure we need this method specifically
```suggestion
    1.week.ago.beginning_of_day
```

or 
```suggestion
    1.week.ago.midnight
```

both options would return the same results.

Alternatively, reusing `start_of_today` method

```suggestion
    start_of_today - 7.days
```

Reply:
I've gone with the last one, as it removes the most gubbins